### ISSUE TITLE
Unknown column order.mollie_refunds.version_id

### ISSUE THREAD

**Issue by stillartem** [mollie/Shopware6#518]
After upgrading to version 3.5.0 shop throw this exception on order deletion action

Precondition
1. Upgrade or install the mollie plugin with version 3.5.0
2. Shopware core version 6.4.20
3. Create sandbox order with the payment method "credit card mollie"
4. Try to delete the order

Actual Result
Order cannot be deleted. Error:
```
An exception occurred while executing 'SELECT LOWER(HEX(`order.mollie_refunds`.`id`)) as id FROM `order` `order` LEFT JOIN `mollie_refund` `order.mollie_refunds` ON `order`.`id` = `order.mollie_refunds`.`order_id` AND `order`.version_id = `order.mollie_refunds`.version_id WHERE (`order.mollie_refunds`.`id` IS NOT NULL) AND (`order`.`version_id` = ?) AND ((`order`.`id` = ? AND `order`.`version_id` = ?))' with params ..."]
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'order.mollie_refunds.version_id'
```
Expected Result
Order successfully deleted

**Comment by boxblinkracer** [comment]
hi there

oh...when deleting the order...that was indeed not tested.
ill make sure it will be fixed

thank you

**Comment by jamieyouellBuro26** [comment]
Thanks for the fix. When is the next release?

**Comment by boxblinkracer** [comment]
probably already next week
we try to release it as soon as possible :)

**Comment by boxblinkracer** [comment]
fixed in v3.6.0, thx for your help
